prow/plugins/override/override.go

```diff
@@ -147,20 +155,24 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent)
 
 	mat := overrideRe.FindAllStringSubmatch(e.Body, -1)
 	if len(mat) == 0 {
-		return nil
-	}
-
-	overrides := sets.String{}
-
-	for _, m := range mat {
-		overrides.Insert(m[1])
+		return nil // no /override commands given in the comment
 	}
 
 	org := e.Repo.Owner.Login
 	repo := e.Repo.Name
 	number := e.Number
 	user := e.User.Login
 
+	overrides := sets.NewString()
+	for _, m := range mat {
+		if m[1] == "" {
+			resp := "/override requires a failed status context to operate on, but none was given"
+			log.Warn(resp)
```

Comment:
Can we switch this warning (and the one below) to debug level? There isn't any unexpected behavior here, everything is WAI, this is entirely user error and doesn't require attention from a Prow admin.